ms/MSOper/MSMetaData.h

```diff
@@ -85,12 +85,13 @@ class MSMetaData {
         // the subscan and that spwID
         map<uInt, Quantity> meanInterval;
         Quantity meanExposureTime;
-        uInt nrows;
         std::set<uInt> spws;
         // number of rows for each spectral window
         map<uInt, uInt> spwNRows;
         std::set<Int> stateIDs;
-        std::map<Double, TimeStampProperties> timeProps;
+        map<Double, TimeStampProperties> timeProps;
```

Comment:
it needs to be std::map, that you can use it without is an error, std namespace should not be set default in headers